bundle-configs/salesforcedx-vscode-apex-replay-debugger/esbuild.config.js

```diff
@@ -15,8 +15,8 @@ const sharedConfig = {
   platform: 'node',
   external: [
     'vscode',
-    // '@salesforce/core-bundle',
-    'applicationinsights'
+    'applicationinsights',
+    'jsonpath'
```

Comment:
it is easy to lose track of these indirect dependencies in the long run. How can we keep track of which dependencies we are externalizing and why?
For e.g. say in future we stop relying on bfj module for some reason, this dependency will still be externalized at the extension level. How can we avoid this situation?